.gitignore

```diff
@@ -34,4 +34,8 @@ yarn-error.log*
 notes.txt
 
 # tests
-/coverage
\ No newline at end of file
+/coverage
+
+# generated ajv schema
+/src/__generated__/*
```

Comment:
Unless you feel strongly about it, I'd rather this be put in something like `/src/ajv` or `/src/schema` or `/src/validators`, that better describes what it is. This is what is done with `/src/types/v3` and `/src/abis/types` (at the top of the file). I also don't want a separate pattern here for file generation.

Reply:
i do feel like it is useful to know which code is generated vs authored. maybe a "lib" folder is better for that? the more practical reason is for lint/type ignores imo, but also i think you don't necessarily want to read generated code

Reply:
i'd be willing to move all of that code to a __generated__ file path or lib if you agree

Reply:
Would you be willing to put this into `/src/validators`, and then follow-up by moving _all_ generated code to a `__generated` directory?